### PR TITLE
74 content development

### LINKED ISSUE #74: Content development for the Bulletin
Adding and edting content for the bulletins

### PR DESCRIPTION
**Pull request by cosola**
Added content and changed background colors
closes #74

### CODE REVIEW

**Comment by misscs** [comment]
Screen Shot 2020-04-17 at 14 26 14

git commit messages should be short and ideally fit completely without over running the character space provided. You can see how your commit title runs off. git has a description field. 

you should get in the habit of using `git commit`. It will open an editor. You add you commit title and a return so there is a blank space. what you add after will appear as the description. that is where you include more details about what you are committing.

File: app/assets/styles/bulletin.css
Comment (misscs) [inline app/assets/styles/bulletin.css]: We should not leave code that is commented out in the repo. Can you please remove line 32.

File: app/assets/styles/bulletin.css
Comment (misscs) [inline app/assets/styles/bulletin.css]: Same comment above applies here. Please remove lines 64 and 68

File: app/assets/styles/bulletin.css
Comment (misscs) [inline app/assets/styles/bulletin.css]: If we are using BEM then we should target classes and not use descendant selectors. It can lead to problems down the line. This is an issue we have in the first newsletter. Please target a class directly.

File: app/assets/styles/bulletin.css
Comment (misscs) [inline app/assets/styles/bulletin.css]: Remove line 125

File: app/assets/styles/bulletin.css
Comment (misscs) [inline app/assets/styles/bulletin.css:132]: @Morban15 @cosola I am noticing an inconsistency in the file when it comes to colors. Some are hex and others are RGB. We should pick a format and go from there. I also think we should start using CSS variables to store the colors at the top of the file that way we can easily just reference them with a variable. It also facilitates changing colors much easier. 

My vote is HEX or HSLA. I don't find RGB to be a helpful color format. HEX is understood, HSLA is my favorite because it is much easier to manipulate colors in a very easy to learn format. If you are not familiar with HSLA, here are some articles

* https://css-tricks.com/yay-for-hsla/
* https://css-tricks.com/hsl-hsla-is-great-for-programmatic-color-control/

File: app/assets/styles/bulletin.css
Comment (misscs) [inline app/assets/styles/bulletin.css]: This code should be removed

File: app/assets/styles/bulletin.css
Comment (misscs) [inline app/assets/styles/bulletin.css]: Remove commented out code.

File: app/assets/styles/bulletin.css
Comment (misscs) [inline app/assets/styles/bulletin.css]: We should avoid styling to an ID. Can you replace this with a class?

File: bulletin/index.html
Comment (misscs) [inline bulletin/index.html:45]: We don't need to use an HTML entity when we use the proper encoding `<meta charset="UTF-8">`.

File: bulletin/index.html
Comment (misscs) [inline bulletin/index.html]: > The MACILE Summer Academy

Add MACILE

File: bulletin/index.html
Comment (misscs) [inline bulletin/index.html]: Change content to: 

The 2007 MACILE Summer Academy lasted for three weeks and only include mathematics.

File: bulletin/index.html
Comment (misscs) [inline bulletin/index.html:64]: What do you mean by "people that made it possible in the field"...in the field of what?

> Itabo elementary school director(far left), 6th history and Spanish teacher (far right), and community leaders. They visited the schools, selected students, provided the space, and assisted with logistics

I think we can improve this to make it clearer. To help me understand what you are trying to convey, I need to ask a few questions:

- Why did the visit the schools? What schools did they visit?
- They selected students for what—to apply to the program?
- Provided the space...for the summer 2007 program?

File: bulletin/index.html
Comment (misscs) [inline bulletin/index.html]: Same here, should always be MACILE Summer Academy

Comment (misscs) [reply bulletin/index.html]: change 

> Far right, Yahaira Pinales, Dominican teacher that recruited the students.

to 

Yahaira Pinales (far right) Dominican teacher that recruited the students.

We should keep consistent and put the position/location of people in photos as a parenthetical comment after the name. This tends to be the pattern on news sites / across the web.

File: bulletin/index.html
Comment (misscs) [inline bulletin/index.html]: spelling correction -> Nurture

ps. vsCode has a spell check feature ++

File: bulletin/index.html
Comment (misscs) [inline bulletin/index.html]: spelling correction

* access
* stimulating

File: bulletin/index.html
Comment (misscs) [inline bulletin/index.html]: It is always best to write in the active voice. Also, I think a more conversational tone would be nicer, makes it easier to read. Here's my suggestion: 

Based on the observations in 2007, we adjusted the 2008 MACILE Summer Academy. It was extended to five weeks and we redesigned the curriculum to include engineering, technology, Spanish literacy, in addition to math. We also added activities—chess, field trips, and guest speakers.

File: bulletin/index.html
Comment (misscs) [inline bulletin/index.html]: should be MACILE Summer Academy 2008

File: bulletin/index.html
Comment (misscs) [inline bulletin/index.html:85]: Here you spell out "thirty-five". above you used numerals (47 students). The document should remain consistent. Please pick a style format and update.

File: bulletin/index.html
Comment (misscs) [inline bulletin/index.html]: This is also in the passive voice. 

We extended the school day from 12-4PM and provided students with a healthy lunch in addition to breakfast. We gauged the students' mathematical and linguistic progress through a math test, and an evaluation questionnaire completed at the end of the program.

File: bulletin/index.html
Comment (misscs) [inline bulletin/index.html]: Rework the sentence and corrected various spelling errors. 

MACILE Summer Academy July 2007-2008. Conducted at the Itabo Elementry School in Itabo, San Cristóbal.

File: bulletin/index.html
Comment (misscs) [inline bulletin/index.html]: should be -> proudly

File: bulletin/index.html
Comment (misscs) [inline bulletin/index.html]: Do you need to capitalize Indios?

File: bulletin/index.html
Comment (misscs) [inline bulletin/index.html:104]: spelling + grammar

* folklore
* written by

Comment (misscs) [reply bulletin/index.html:104]: > Los indios de Quisqueya, the accompaning history book and foundation of the MACILE curriculum, was donated by the Tavares Foundation. The author, Juan Tómas Tavares Kelnes, permited its use.

Los Indios de Quisqueya and the accompanying history book form the foundation of the MACILE (history or Spanish) curriculum. We thank the Tavares Foundation for the generous donation and the author Juan Tómas Tavares Kelnes for the permission to use his work. 

- I recommend that you qualify curriculum otherwise it reads that these books form the foundation of the entire MACILE curriculum. 
- If you don't want to use the re-write, there are various spelling errors in the original text

File: bulletin/index.html
Comment (misscs) [inline bulletin/index.html]: * drum
* and learning Spanish

Comment (misscs) [reply bulletin/index.html]: If this content is supposed to be a caption for the image it would be more semantic to use `<figure>` and `<figcaption>`. It would also improve the experience for screen readers.

File: bulletin/index.html
Comment (misscs) [inline bulletin/index.html:117]: This doesn't read so clearly with the commas. I think using `—` would be much clearer because it wouldn't confuse what it applies to.

File: bulletin/index.html
Comment (misscs) [inline bulletin/index.html]: * leaders
* the job market

Hmm..."opportunities to be successful" reads oddly. What is it that you want to say here?

File: bulletin/index.html
Comment (misscs) [inline bulletin/index.html]: * participating in a two week long math training workshop: Improving Learning of Basic Algebra

File: bulletin/index.html
Comment (misscs) [inline bulletin/index.html]: - MACILE Scholarship Program (proper noun, should be capitalized)
- is merit based and for talented young people from ...

What do you want to say here

> and be better prepared to pursue higher learning in STEM fields.

that the program prepares them to pursue higher learning or that the students will be better prepared?

File: bulletin/index.html
Comment (misscs) [inline bulletin/index.html:141]: I don't agree with the usage of `article` tags here. `figure` and `figcaption` is much more semantic. These don't really represent an "`article` which one would expect to have more content. A `div` would be less semantic, but still a better choice.

**Review by misscs — CHANGES_REQUESTED** [review]
@cosola I finished the review. Mostly grammatical corrections and spelling corrections. Let me know if you have any questions. This story is blocking another story, so it would be best if you can address the code review before working on other features. Thanks!

File: app/assets/styles/bulletin.css
Comment (cosola) [reply app/assets/styles/bulletin.css]: #74 
All commented out lines of code remove.

File: app/assets/styles/bulletin.css
Comment (cosola) [reply app/assets/styles/bulletin.css]: Completed

File: app/assets/styles/bulletin.css
Comment (cosola) [reply app/assets/styles/bulletin.css]: Done

File: bulletin/index.html
Comment (cosola) [reply bulletin/index.html:141]: I absolutely agree with you,  article tags is not semantic. It is more appropriate to use div throughout the bulletin.

File: app/assets/styles/bulletin.css
Comment (cosola) [reply app/assets/styles/bulletin.css]: Done

File: bulletin/index.html
Comment (cosola) [reply bulletin/index.html:45]: I understand, but certain redundancy doesn't hurt.

File: bulletin/index.html
Comment (cosola) [reply bulletin/index.html:64]: - [x] The teachers visited the schools to talk about MACILE and the summer program
- [x] They visited Yogo Yogo, Itabo, Siglo XX1 in El 18, and Haina.
- [x] The teachers recommended the students and essentially all were accepted. There was no selection mechanism in place.
- [x] The director and teachers of Itabo Elementary welcome MACILE in the school. They provided the space and were very helpful.

File: bulletin/index.html
Comment (cosola) [reply bulletin/index.html]: Thanks. Will check it out.

File: bulletin/index.html
Comment (misscs) [reply bulletin/index.html:141]: Well, it may not just be `div`'s. Some of these can use `figure` and `figcaption`. We should review the whole document. So if you want to wait to resolve this until #87, I'm fine with letting it go. We have a story that is an assurance it will get fixed.

File: app/assets/styles/bulletin.css
Comment (misscs) [reply app/assets/styles/bulletin.css]: Great, thank you!

**Comment by misscs** [comment]
Good work @cosola with single purpose commits! One last bit of feedback is to Title case your sentences. Otherwise, great improvements.

File: bulletin/index.html
Comment (cosola) [reply bulletin/index.html:117]: Good observation. Re-wrote. I think it's clearer now.

File: bulletin/index.html
Comment (cosola) [reply bulletin/index.html]: This section is incomplete. Will finalize latter.

File: app/assets/styles/bulletin.css
Comment (misscs) [inline app/assets/styles/bulletin.css:102]: This class is named blue but the color is white. That's very strange. 

I don't want to hold up this PR any longer, but we should decide how to handle colors. I'm okay mixing BEM and atomic classes, however, I think the classes should match their names. Take a look at the Tachyons project if you want to see a functionall/atomic css library and how their color classes match.

File: bulletin/index.html
Comment (misscs) [reply bulletin/index.html:64]: Okay. Since you opened #88, I think we can work the changes out there.

File: bulletin/index.html
Comment (misscs) [inline bulletin/index.html:124]: Again, keep your commits single purpose. The commit message says "complete content for the scholarship section" but the commit includes

- changes to class
- changes to html that does not correspond to the scholarship section

You should get in the habit of using `git add -p` to selectively stage the commits that you want. VSCode has a lovely git plugin that lets you easily pick the sections you want to add to your commit.

File: bulletin/index.html
Comment (misscs) [inline bulletin/index.html]: You introduced various regressions. It needs to be: 

- the job market
- scholarship programs

File: bulletin/index.html
Comment (misscs) [inline bulletin/index.html:136]: _It's open to students_ not opened

File: bulletin/index.html
Comment (misscs) [reply bulletin/index.html:104]: @cosola Whoops, I think you forgot to include these corrections.

File: bulletin/index.html
Comment (misscs) [reply bulletin/index.html:117]: It's better, but still not so clear. It's a very positive sentence that is strong on its own, it doesn't benefit from the "as Einstein admonished". I think it creates a negative feel. Also the way it is written, it isn't clear what he admonished, you need to clearly separate that clause. 

MACILE strives to expand the young people's horizons, nourishing their innate curiosity and wonderful imagination; rather than discouraging them, as Einstein admonished.

or 

MACILE strives to expand the young people's horizons, nourishing their innate curiosity and wonderful imagination, rather than discouraging them, as Einstein admonished.

Comment (misscs) [reply bulletin/index.html:117]: But my vote is to remove it. It doesn't add anything.  Let sentence stand on its own. It is more powerful that way.

File: bulletin/index.html
Comment (misscs) [inline bulletin/index.html:121]: Here you made the fix and then you reverted it in the next commit 🤦‍♀️

Comment (misscs) [reply bulletin/index.html:121]: "for success" .... in what...it was better before.

**Review by misscs — CHANGES_REQUESTED** [review]
Do you have time to fix the section of _Los Indios de Quiskeya_? I think that's really the section worth correcting. We can open up a story to do another pass at correcting the content. This PR introduced various regressions. I want to get this merged because it is blocked #58. The content is in a decent enough position to complete that story even with the errors.

File: app/assets/styles/bulletin.css
Comment (cosola) [reply app/assets/styles/bulletin.css:132]: I suggest all students read these articles. Like the idea of using CSS variables to store colors.

File: bulletin/index.html
Comment (cosola) [reply bulletin/index.html:117]: I feel a bit uncomfortable taking implicitly credit for Einstein's view.  But, it might be OK to remove his name.

Consider this alternative:
Following Einstein's advice, MACILE strives to expand the young people's horizons, nourishing their innate curiosity and wonderful imagination, rather than discouraging them.

File: app/assets/styles/bulletin.css
Comment (cosola) [reply app/assets/styles/bulletin.css:102]: Blue refers to the background color.

I agree we need better names for the classes, and suggest we create a new issues to improve the HTML semantic and CSS.

Please close the PR.  I want to move to the next step, if possible.

File: bulletin/index.html
Comment (cosola) [reply bulletin/index.html:124]: Thanks for the tip. Will keep in this mind. Important to maintain the consistency of the process throughout.

File: bulletin/index.html
Comment (cosola) [reply bulletin/index.html]: Done

File: bulletin/index.html
Comment (cosola) [reply bulletin/index.html:136]: Done.

File: bulletin/index.html
Comment (cosola) [reply bulletin/index.html:121]: Done.

**Comment by cosola** [comment]
The section "Los Indios de Quisqueya" will be re-redacted latter. I want to review the Spanish book before writing the content for this section.  This is a placeholder.

**Review by misscs — APPROVED** [review]
We will approve and merge this PR now. We need to open separate stories for editing and addressing the content changes.

**Pull request merged**
